### ISSUE TITLE
reproduce your work

### ISSUE THREAD

**Issue by jianwang91** [wangbo-zhao/WSVSOD#4]
Thanks for your great work.However, I got trouble reproducing your work.
1. the images are jpg format while the gt, mask, etc. are png format. But the load of the dataset didn't transform it; thus I got a file not found error. After I fixed it myself, I found the pretrained process use the optical flow, which I think it's not reasonable.So I wonder if you uploaded the wrong version of your code.
2 The fine-tune process needs 8 file directory, I wonder how can I get fixation_maps, fixation_maps_smoothed, flow and grey? Could you please provide the  files, since I didn't find them in dataset.rar in the link you provided of BaiduNetDisk so that we can reproduce your wonderful work more easily?

Thanks.

**Comment by wangbo-zhao** [comment]
Thanks for your interest.
1. In fact, we adopt optical flow maps generated from static images. This
process is described in " Implementation details " in our paper.
2. fixation_maps and fixation_maps_smoothed are directly downloaded from
the datasets that we talked about in our paper. You can use any optical
flow estimation method to generate the flow. The gery can be generated from
original RGB frames.

muchengxue0911 ***@***.***> 于2022年7月21日周四 16:28写道：

**Comment by jianwang91** [comment]
Thanks for your patient answer, I'll try it.

**Comment by jianwang91** [comment]
After generating the optial flow, another error appeared:
too many values to unpack (expected 5) in line 272 in your model.py file, which is because you want to unpack 5 tensors while the rgb_feature is a 4 dimentions tensor.
Thus, once again, can you recheck your code and uploaded a runnable version  of your great work?
Thanks!

**Comment by jianwang91** [comment]
And what's more, I want to finetune the model directly using your pretrained model, however, I have difficulties in the fiation_maps and fixation_maps_smoothed files. I noticed that you mentioned reference 37 provided the fixation map of DAVIS, but the link in his github is invalid. Moreover, where can I find the fixation_map and smoothed files for DAVSOD? Can you kindly upload them to your BaiduNetDisk? Thanks a lot.

**Comment by wangbo-zhao** [comment]
Sorry for confusing you. I remember that I do not use fiation_maps and fixation_maps_smoothed files during training and finetune in fact. Please try commenting codes related to fixation_map and fixation_map_smoothed in the dataset. I will fix this bug later.
![image](https://user-images.githubusercontent.com/56866854/182058899-1840c4c6-1707-4f6f-9edb-8d5c121c6f43.png)

**Comment by jianwang91** [comment]
OK，Thanks a lot. I will have a  try ~

**Comment by wangbo-zhao** [comment]
I would like to know whether you solve the problem "too many values to unpack (expected 5) in line 272 in your model.py file, which is because you want to unpack 5 tensors while the rgb_feature is a 4 dimentions tensor". Since I do not have the environment to run this code now, please tell me the bug if you find it.

**Comment by jianwang91** [comment]
> 

No， I'm just trying the finetune period now. I'll let you know when it's done.

**Comment by jianwang91** [comment]
Sorry to bother you again. Could you please upload your optical flow images? I use a network to generate the optical flow, but the results are lower than your paper.

**Comment by wangbo-zhao** [comment]
Sorry. I do not have these optical flow images. Because I have graduated
from NWPU.  What network do you use to generate the optical flow?

muchengxue0911 ***@***.***> 于2022年10月21日周五 16:35写道：

**Comment by jianwang91** [comment]
I use a pytorch version of pwcnet.

**Comment by wangbo-zhao** [comment]
Is the difference between your results and our results large?

muchengxue0911 ***@***.***> 于2022年10月21日周五 18:39写道：

**Comment by jianwang91** [comment]
Yes，S and F is about 3%  lower in DAVIS and ViSal,  1% at other datasets
